packages/office-ui-fabric-react/src/common/_semanticSlots.scss

```diff
@@ -40,7 +40,7 @@ $buttonTextCheckedHoveredColor: "[theme:buttonTextCheckedHovered, default: #0000
 
 /* Menus */
 $menuItemBackgroundHoveredColor: "[theme:menuItemBackgroundHovered, default: #f8f8f8]";
-$menuItemBackgroundCheckedColor: "[theme:menuItemBackgroundChecked, default: #d0d0d0]";
+$menuItemBackgroundCheckedColor: "[theme:menuItemBackgroundChecked, default: #eaeaea]";
```

Comment:
i thought we were going to set it to no background?

Reply:
Can you elaborate?

Reply:
@lynamemi I thought the design was to have it have a white, blank background, but ignore me if that's the case.

Reply:
nevermind, I thought this was something else